ee/maintained-apps/ingesters/homebrew/external_refs/main.go

```diff
@@ -0,0 +1,25 @@
+package externalrefs
+
+import (
+	"fmt"
+
+	maintained_apps "github.com/fleetdm/fleet/v4/ee/maintained-apps"
+)
+
+var Funcs = map[string][]func(*maintained_apps.FMAManifestApp) (*maintained_apps.FMAManifestApp, error){
+	"microsoft-word/darwin":  {MicrosoftVersionFromReleaseNotes},
+	"microsoft-excel/darwin": {MicrosoftVersionFromReleaseNotes},
+}
```

Comment:
I did prefer transformations on each attribute because it allowed for smaller methods and more easily testable logic. Whereas one big func to "modify microsoft apps" becomes unwieldy. So split the difference with a slice of func to be called on the app?